runtime/src/accounts_db.rs

```diff
@@ -4015,13 +4047,22 @@ impl AccountsDB {
         hashes: &[Hash],
         is_cached_store: bool,
     ) {
+        // This path comes from a store to a non-frozen slot.
```

Comment:
heh , maybe we should call this `store_accounts_non_frozen()` to make sure nobody calls this/or is wary of calling this from another path in the future.